symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/pom.xml

```diff
@@ -42,7 +42,37 @@
             <groupId>org.apiguardian</groupId>
             <artifactId>apiguardian-api</artifactId>
         </dependency>
-
     </dependencies>
 
+    <build>
```

Comment:
isn't already defined in the parent pom?

Reply:
No but I guess we should